test/Spec/TestContract/ValidateTimeRange.hs

```diff
@@ -109,30 +110,28 @@ typedValidator =
     $$(PlutusTx.compile [||mkValidator||])
     $$(PlutusTx.compile [||wrap||])
   where
-    wrap = Validators.wrapValidator @() @TimeRedeemer
+    wrap = mkUntypedValidator @() @TimeRedeemer
 
 validator :: Validator
 validator = Validators.validatorScript typedValidator
 
 validatorAddr :: Address
-validatorAddr = scriptAddress validator
+validatorAddr = mkValidatorAddress validator
 
 ------------------------------------------
 failingTimeContract :: Contract () EmptySchema Text Hask.String
 failingTimeContract = do
   startTime <- Contract.currentTime
-  let noOfSlots = 10
-      slotLen = 1000
-      timeDiff = POSIXTime (noOfSlots * slotLen)
+  let timeDiff = POSIXTime 5_000
```

Comment:
Note: if we decide to go back to `nOfSlots * slotLen` representation, we will need to get real slot length from cluster, as it can vary (although it's not too clear yet how to set it after update to newer version of `cardano-wallet` framework; default is 0.2 sec).